scripts/add_plugin.py

```diff
@@ -45,9 +45,7 @@ def main(*, plugin: str, url: str, prefix_code: str) -> None:
     fn rules(rule_code: Rule, path: &Path) -> Result<()> {
         let snapshot = format!("{}_{}", rule_code.as_ref(), path.to_string_lossy());
         let diagnostics =test_path(
-            Path::new("./resources/test/fixtures/%s")
```

Comment:
Was the inclusion of `./resources/test/fixtures/` here wrong before? (I think so?)

Reply:
Uh, I didn't notice that I removed the `test/fixtures` too. I just searched for the committed code and inferred the pattern from there. 

https://github.com/charliermarsh/ruff/blob/c6887fa4d35d67971930b7d681f70bb561423ef5/crates/ruff/src/rules/tryceratops/mod.rs#L26-L29

Reply:
Yeah since `test_path` was introduced, I think the pattern we have now is correct. (IIRC, we do test these scripts on CI, but only that the code still compiles after running them, which wouldn't catch this issue.)